### DISCUSSION TITLE
Icon request: chair-and-desk

### DISCUSSION THREAD

**Discussion by federicoalvarez-github** [FortAwesome/Font-Awesome#21215] [Icon Requests]
Hi there! Could you please make an icon of a chair and a desk? Just standard chair and desk. The chair would be like an "h", and the desk like an "n" but with a little bit more table on it. Hope I made myself clear! Best regards and congratulations on such a wonderful job. (I'm a kickstarter backer!). Federico.

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂
